Libraries/acdriver/Headers/acdriver/Extensions.h

```diff
@@ -0,0 +1,84 @@
+/**
+ Copyright (c) 2015-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#ifndef __acdriver_Extensions_h
+#define __acdriver_Extensions_h
+
+#include <acdriver/Result.h>
+#include <car/Rendition.h>
+
+#include <algorithm>
+#include <ext/optional>
+#include <string>
+#include <unordered_set>
+#include <utility>
+#include <vector>
+
+namespace acdriver {
+
+/*
+ * Contains non-standard related logic
+ */
+class Extensions {
```

Comment:
I'm confused why there's a class (a namespace, really) called Extensions. Why group all extensions together? What do different (future) extensions have to do with each other?

Reply:
The term "extension" confused me when reading the code in ImageSet.cpp. "Extension" can refer to a *file* extension (like `.png`).

Reply:
I just wanted to make it clear what's standard behavior (compatible with Apple's actool) and what's extra feature this provides.
Initially I mixed all together, but @Ktwu asked me to move this logic to its own file as much as possible. I think it's become much better like this.
I even used a class instead of namespace to make it clear when we are using that. (e.g. you can not do `using acdriver::Extensions::ImageTypeFromFileExtension` and then use `ImageTypeFromFileExtension` alone, you need to always call like `Extensions::ImageTypeFromFileExtension`)

Yeah, if you have another suggestion of name, that's why I tried to always use "fileExtension" when applicable.

Reply:
I can do `Extensions` -> `NonStandard` with everything in this diff. What do you think @strager and @Ktwu ?